public/app/features/profile/UserPreferences.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { PureComponent } from 'react';
+import { hot } from 'react-hot-loader';
+import { connect } from 'react-redux';
+import { StoreState, OrgUser } from 'app/types';
+import { loadUser, clearUser } from 'app/core/components/UserEdit/state/actions';
+import { getNavModel } from 'app/core/selectors/navModel';
+import { NavModel } from '@grafana/data';
+import Page from 'app/core/components/Page/Page';
+import UserProfile from 'app/core/components/UserEdit/UserProfile';
+import UserSessions from 'app/core/components/UserEdit/UserSessions';
+import UserTeams from 'app/core/components/UserEdit/UserTeams';
+import SharedPreferences from 'app/core/components/SharedPreferences/SharedPreferences';
+
+export interface Props {
+  navModel: NavModel;
+  user: OrgUser;
+  loadUser: typeof loadUser;
+  clearUser: typeof clearUser;
+}
+
+export class UserPreferences extends PureComponent<Props> {
+  async componentDidMount() {
+    await this.props.loadUser();
+  }
+
+  componentWillUnmount() {
+    this.props.clearUser();
+  }
+
+  render() {
+    const { navModel, user } = this.props;
+    const isLoading = Object.keys(user).length === 0;
+    return (
+      <Page navModel={navModel}>
+        <Page.Contents isLoading={isLoading}>
+          {!isLoading && <UserProfile />}
+          <SharedPreferences resourceUri="user" />
+          <UserTeams />
```

Comment:
Missing UserOrganizations here (in view only mode, ie no ability to add or modify role), this was here in the angular version.